htlcswitch/interfaces.go

```diff
@@ -30,7 +30,7 @@ type InvoiceDatabase interface {
 //       |
 //       | (Switch)		     (Switch)		       (Switch)
 //       |  Alice <-- channel link --> Bob <-- channel link --> Carol
-//	 |
+//       |
```

Comment:
Can you revert this? Thanks!

Reply:
I replaced a tab with spaces there. On my editor it showed wrong... Was that intentional?

Reply:
(All the other `|`'s are indented with spaces while that lone one was the only one indented by a tab..)